### DISCUSSION TITLE
How to change update interval

### DISCUSSION THREAD

**Discussion by Tokn59** [SBFspot/SBFspot#706] [General]
Hi, THANK YOU for this great tool but how can I change the default 5 min update to f.i. 1 minute ?

**Comment by sbf-** [comment]
Change the crontab:

```
*/1 6-22 * * * /usr/local/bin/sbfspot.3/daydata
```

**Reply by Tokn59** [reply]
Thank You !!!!

**Comment by chzetzl** [comment]
Is this still working with newest release?
iOBroker refreshes only every 5 minutes and can't be changed to lower than 5 minutes.
